metrics.md

```diff
@@ -162,23 +158,35 @@ Numerator resource inclusion:
+| Condition          | code                      | http://hl7.org/fhir/sid/icd-9-cm or<br>http://hl7.org/fhir/sid/icd-10-cm or<br>http://snomed.info/sct                                                                                                                            |
+| Device             | type                      | http://snomed.info/sct                                                                                                                                                                                                           |
+| DiagnosticReport   | code                      | http://loinc.org                                                                                                                                                                                                                 |
+| DocumentReference  | type                      | http://loinc.org                                                                                                                                                                                                                 |
+| Immunization       | vaccineCode               | http://hl7.org/fhir/sid/cvx                                                                                                                                                                                                      |
+| Medication         | code                      | http://www.nlm.nih.gov/research/umls/rxnorm                                                                                                                                                                                      |
+| MedicationRequest  | medicationCodeableConcept | http://www.nlm.nih.gov/research/umls/rxnorm                                                                                                                                                                                      |
+| Observation        | code                      | http://loinc.org                                                                                                                                                                                                                 |
+| Observation        | valueCodeableConcept      | http://snomed.info/sct                                                                                                                                                                                                           |
+| Procedure          | code                      | http://ada.org/cdt or <br>http://loinc.org or<br>http://snomed.info/sct or<br>http://www.ama-assn.org/go/cpt or<br>http://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets or<br>http://www.cms.gov/Medicare/Coding/ICD10 |
 
 Parameters:
```

Comment:
I'd like to leave parameters alone for now - I think a follow-up PR can add some of the nitty gritty details for at least the metrics I've got some experience with: suggested column names, whether it's suitable for CUBE or just an aggregate table, and suggested parameters.

And maybe mark that all as "implementation guidance", rather than mandatory.

But we can chew on that in a second PR.

Reply:
That works for me. I'd like to update the root level readme a bit more too, but will take a pass at that in another PR as well.